source/_integrations/dhcp.markdown

```diff
@@ -0,0 +1,23 @@
+---
+title: DHCP Discovery
```

Comment:
```suggestion
title: DHCP
```

To match the naming of the other integrations.

Reply:
I don't think this suggestion is appropriate.

I think it helps Discovery is added here, mainly to prevent people of thinking this is a DHCP server. As for the documentation, it should match the manifest (or else it is overwritten on release anyway as codebase syncs up with the docs).

So if this is changed, it should be changed in the manifest, not the documentation. However, as said, I don't agree on changing it.